ruff.toml

```diff
@@ -6,15 +6,21 @@ target-version = "py310"
 
 extend-exclude = ["*.gypi"]
 
+fix = true
+
+unsafe-fixes = true
+
 # Enable flake8-bugbear (`B`) rules.
 # https://beta.ruff.rs/docs/configuration/#using-rufftoml
 [lint]
+fixable = ["ALL"]
```

Comment:
These changes should be reverted

We would not want to set `fix = true` since it disables reporting on unfixed errors.
Enabling `unsafe-fixes` is not desirable.
`fixable = ["ALL"]` is already the default.

Reply:
reverted